### PR TITLE
Speech pv

### PR DESCRIPTION
**Pull request by dizcology**
Do not merge until the updated client library has been published and tested.

### CODE REVIEW

File: speech/cloud-client/transcribe_with_metadata_test.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# limitations under the License.
+
+import os
+
+import transcribe_with_metadata
+
+RESOURCES = os.path.join(os.path.dirname(__file__), 'resources')
+
+
+def test_transcribe_file_original_media_type_video(capsys):
```
Comment (jmdobry) [inline speech/cloud-client/transcribe_with_metadata_test.py]: Shouldn't this be renamed to `test_transcribe_file_with_metadata`?

File: speech/cloud-client/transcribe_with_metadata_test.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    transcribe_with_metadata.transcribe_file_with_metadata(
+        os.path.join(RESOURCES, 'Google_Gnome.wav'))
+    out, _ = capsys.readouterr()
+
+    assert 'you do keep doing that' in out
+    assert 'organic matter and will return' in out
+
+
+def test_transcribe_gcs_original_media_type_video(capsys):
```
Comment (jmdobry) [inline speech/cloud-client/transcribe_with_metadata_test.py]: Shouldn't this be renamed to `test_transcribe_gcs_with_metadata`?

**Comment by jonparrott** [comment]
@dizcology is this PR still relevant?

**Comment by dizcology** [comment]
Sorry for the delay.  This PR is no longer relevant.  Closing it now and I will create a separate PR soon.

**Pull request closed without merging**